commit 4df9a681da04db311d422976d96b2db5f08a3765 (refs/changes/41/2841/1)
Author: Orlando Vazquez <ovazquez@gmail.com>
Date:   2017-10-23T02:12:20-07:00 (2 years ago)
    
    CNAPI-725 test-task test failing because of a change introduced in CNAPI-724

diff --git a/lib/endpoints/tasks.js b/lib/endpoints/tasks.js
index 1aa5f58..ba03c14 100644
--- a/lib/endpoints/tasks.js
+++ b/lib/endpoints/tasks.js
@@ -85,11 +85,7 @@ Task.wait = function handlerTaskWait(req, res, next) {
 
         wait(function (err) {
             if (err) {
-                req.log.error(err, 'error waiting on task');
-
-                next(new restify.InternalError(
-                    'error waiting on task'));
-                return;
+                req.log.error(err, 'task we waited on had an error');
             }
 
             get(function () {
